arcticpathing/main.py

```diff
@@ -0,0 +1,21 @@
+from arcticpathing import utils, data, pathing, constants

```

Comment:
If you find yourself with a long line of imports, note that it's valid to wrap the imports in parens and break them onto new lines, e.g.,

```
from arcticpathing import (
    utils,
    data,
    pathing,
    constants,
)
```